docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -234,14 +260,14 @@ To provide a custom location for the Flink configuration files, you can
 * **either mount a volume** with the custom configuration files to this path `/opt/flink/conf` when you run the Flink image:
 
     ```sh
-    docker run \
+    $ docker run \
         --mount type=bind,src=/host/path/to/custom/conf,target=/opt/flink/conf \
         flink:{% if site.is_stable %}{{site.version}}-scala{{site.scala_version_suffix}}{% else %}latest{% endif %} <jobmanager|standalone-job|taskmanager>
     ```
 
 * or add them to your **custom Flink image**, build and run it:
 
-    *Dockerfile*:
+  *Dockerfile*:
```

Comment:
```suggestion
```
I think we can remove `Dockerfile:` here as it's mentioned in the item above that it's a "custom Flink image"